libs/barista-components/radial-chart/src/radial-chart.ts

```diff
@@ -377,4 +377,17 @@ export class DtRadialChart implements AfterContentInit, OnDestroy {
       series.origin.selectedChange.emit(series.origin.selected);
     }
   }
+
+  /** @internal Toggle the visibility of an element */
+  _toggleLegend(slice: DtRadialChartRenderData): void {
+    // don't allow hiding last element
+    if (
+      this._renderData.filter((node) => node.origin.active).length > 1 ||
+      !slice.origin.active
```

Comment:
I think you don't need the second statement because it would already be false, right?

Reply:
You're right, second condition is not needed

Reply:
No, the condition is needed because if you left only one active legend you would not be able to re activate the rest of them